src/Sql/dbo/Stored Procedures/Organization_Create.sql

```diff
@@ -1,5 +1,5 @@
 ﻿CREATE PROCEDURE [dbo].[Organization_Create]
-    @Id UNIQUEIDENTIFIER,
+    @Id UNIQUEIDENTIFIER OUTPUT,
```

Comment:
Are these changes purely to support the framework and model changes since these are not `IDENTITY` values and are explicitly passed in (code-generated GUID values)?

Reply:
Yes. Since all these methods run through the same `BaseRepository.CreateAsync` method we have to do something to get back IDs that works for both auto incrementing ints and code-generated guids. Applying OUTPUT on everything regardless of type is where I landed vs. something like `if typeof(obj.Id) == typeof(Guid) { do something else }`